From: Antonio Radici <antonio@debian.org>
Date: Thu, 27 Feb 2014 16:06:15 +0100
Subject: use_usr_bin_editor

Even if EDITOR is not set, mutt will always use /usr/bin/editor
(which is set by update-alternatives), rather than falling back
to vi.

Gbp-Pq: Topic debian-specific
---
 init.c | 2 +-
 init.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/init.c b/init.c
index 1401664..e0a0757 100644
--- a/init.c
+++ b/init.c
@@ -3129,7 +3129,7 @@ void mutt_init (int skip_sys_rc, LIST *commands)
   {
     p = getenv ("EDITOR");
     if (!p)
-      p = "vi";
+      p = "/usr/bin/editor";
   }
   Editor = safe_strdup (p);
   Visual = safe_strdup (p);
diff --git a/init.h b/init.h
index 0e4f47f..7768683 100644
--- a/init.h
+++ b/init.h
@@ -686,7 +686,7 @@ struct option_t MuttVars[] = {
   ** .pp
   ** This variable specifies which editor is used by mutt.
   ** It defaults to the value of the \fC$$$VISUAL\fP, or \fC$$$EDITOR\fP, environment
-  ** variable, or to the string ``vi'' if neither of those are set.
+  ** variable, or to the string ``/usr/bin/editor'' if neither of those are set.
   ** .pp
   ** The \fC$$editor\fP string may contain a \fI%s\fP escape, which will be replaced by the name
   ** of the file to be edited.  If the \fI%s\fP escape does not appear in \fC$$editor\fP, a
